classes/system/keepalived/cluster/instance/stacklight_monitor_vip.yml

```diff
@@ -0,0 +1,16 @@
+applications:
+- keepalived
+classes:
+- service.keepalived.support
+parameters:
+  keepalived:
+    cluster:
+      enabled: true
+      instance:
+        stacklight_monitor_vip:
+          address: ${_param:keepalived_vip_address}
+          password: ${_param:keepalived_password}
+          interface: ens4
```

Comment:
just change these to:
${_param:keepalived_stacklight_monitor_vip_address}
${_param:keepalived_stacklight_monitor_vip_interface}
${_param:keepalived_stacklight_monitor_vip_password}
interface default can go to the top of this file